internal/build/image_reaper_test.go

```diff
@@ -0,0 +1,3 @@
+//+build !skipcontainertests
+
+package build
```

Comment:
Is there a reason why this empty file needs to be here?

Reply:
gone!